docs/pages/versions/unversioned/index.md

```diff
@@ -8,35 +8,42 @@ import { Terminal } from '~/ui/components/Snippet';
+The Expo SDK provides access to device and system functionality such as contacts, camera, gyroscope, GPS location, etc., in the form of packages. You can install any Expo SDK package using `expo-cli` with the `expo install` command. For example, three different packages are installed using the following command:
 
 <Terminal cmd={['$ expo install expo-camera expo-contacts expo-sensors']} />
 
-You can import modules from it in your JavaScript code as follows:
+After installing one or more packages, you can import them into your JavaScript code:
 
 ```javascript
 import { Camera } from 'expo-camera';
 import * as Contacts from 'expo-contacts';
 import { Gyroscope } from 'expo-sensors';
 ```
 
-This allows you to write [`Contacts.getContactsAsync()`](sdk/contacts.md#contactsgetcontactsasynccontactquery) and read the contacts from the device, read the gyroscope sensor to detect device movement, or render a Camera view and take photos.
+This allows you to write [`Contacts.getContactsAsync()`](sdk/contacts#contactsgetcontactsasynccontactquery) and read the contacts from the device, read the gyroscope sensor to detect device movement, or start the phone's camera and take photos.
 
 ## These packages work in bare React Native apps too
 
-The easiest way to create a bare React Native app with support for the Expo SDK is `npx create-react-native-app myapp`. If you have an existing app that you would like to add Expo SDK packages to, read about [integrating into existing apps](../../bare/existing-apps.md).
+The easiest way to create a bare React Native app with support for the Expo SDK is by running the command:
+
+<Terminal cmd={[
+'# Create a project named my-app',
+'$ npx create-react-native-app my-app',
+]} cmdCopy="npx create-react-native-app my-app" />
+
+You can install and import any Expo SDK package using the `expo install` command. If you have an existing app that you would like to add Expo SDK packages to, read about [integrating Expo SDK into existing apps](/bare/existing-apps).
 
 ## Each Expo SDK version depends on a React Native version
 
-Every quarter there is a new Expo SDK release that typically updates to the latest stable version of React Native and includes a variety of bugfixes, features and improvements to the Expo SDK.
+Every quarter there is a new Expo SDK release that typically updates to the latest stable version of React Native and includes a variety of bug fixes, features, and improvements to the Expo SDK.
 
 | Expo SDK Version | React Native Version |
 | ---------------- | -------------------- |
-| 45.0.0           |        0.68.2        |
-| 44.0.0           |        0.64.3        |
-| 43.0.0           |        0.64.3        |
-| 42.0.0           |        0.63.3        |
+| 45.0.0           | 0.68.2               |
+| 44.0.0           | 0.64.3               |
+| 43.0.0           | 0.64.3               |
+| 42.0.0           | 0.63.3               |
```

Comment:
```suggestion
| 46.0.0           | 0.69.1               |
| 45.0.0           | 0.68.2               |
| 44.0.0           | 0.64.3               |
| 43.0.0           | 0.64.3               |
```

Add SDK 46, remove SDK 42.

Reply:
Apologies that I missed that didn't read your msg carefully the first time 🙇 
I got confused since SDK 46 is not on prod yet.